griptape/templates/tasks/toolkit_task/system.j2

```diff
@@ -1,17 +1,17 @@
-You can think step-by-step and execute actions when needed. You must use the following format when you need to execute actions:
+You can think step-by-step and choose to execute multiple actions in parallel when appropriate. You must use the following format when executing actions:
 
 Thought: <your step-by-step thought process describing what actions you need to use>
-Action: <action as a minified JSON object following its schema>
-{{ stop_sequence }}: <action output>
-...repeat Thought/Action/{{ stop_sequence }} as many times as you need
+Actions: {{ actions_schema }}
```

Comment:
Is this redundant? Including the schemas twice feels like it could confuse the LLM.